### PR TITLE
Address comments about the IANA section

### PR DESCRIPTION
**Pull request by eranmes**
Addressing comments from Paul Wouters:
* Refer to an RFC for SHA-256.
* Name IANA registries.

### CODE REVIEW

File: draft-ietf-trans-rfc6962-bis.md [hunk 1]
```diff
@@ -1924,19 +1915,21 @@ CachedInformationType Values" registry that was defined in [RFC7924].
...
-IANA is asked to establish a registry of hash algorithm values, initially
+IANA is asked to establish a registry of hash algorithm values, named
+"Hash algorithm numbers for Certificate Transparency logs" initially
 consisting of:
 
 |-------+---------------------|
 | Index | Hash                |
 |-------+---------------------|
-| 0     | SHA-256 [FIPS180-4] |
+| 0     | SHA-256 [RFC4634]   |
```
Comment (robstradling) [inline draft-ietf-trans-rfc6962-bis.md]: Please add RFC4634 as an informational reference.

Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: Done.

Comment (robstradling) [inline draft-ietf-trans-rfc6962-bis.md]: 4634 is Informational, but you've added it as a normative reference.  AIUI, we can only do informative references for Informational RFCs.

Comment (letoams) [inline draft-ietf-trans-rfc6962-bis.md]: yeah please add it as informative reference

Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: Done.

**Comment by eranmes** [comment]
@letoams and @robstradling PTAL.

File: draft-ietf-trans-rfc6962-bis.md [hunk 2]
```diff
@@ -1919,8 +1919,7 @@ CachedInformationType Values" registry that was defined in [RFC7924].
 ## Hash Algorithms    {#hash_algorithms}
 
 IANA is asked to establish a registry of hash algorithm values, named
-"Hash algorithm numbers for Certificate Transparency logs" initially
-consisting of:
+"CT Hash Algorithms" initially consisting of:
```
Comment (robstradling) [inline draft-ietf-trans-rfc6962-bis.md]: There needs to be a comma before "initially".

I think "s/ initially consisting/, that initially consists/" would be better still.

File: draft-ietf-trans-rfc6962-bis.md [hunk 3]
```diff
@@ -1932,8 +1931,7 @@ consisting of:
 ## Signature Algorithms    {#signature_algorithms}
 
 IANA is asked to establish a registry of signature algorithm values, named
-"Signature algorithm numbers for Certificate Transparency logs" initially
-consisting of:
+"CT Signature Algorithms" initially consisting of:
```
Comment (robstradling) [inline draft-ietf-trans-rfc6962-bis.md]: There needs to be a comma before "initially".

I think "s/ initially consisting/, that initially consists/" would be better still.

File: draft-ietf-trans-rfc6962-bis.md [hunk 4]
```diff
@@ -1944,8 +1942,8 @@ consisting of:
 
 ## SCT Extensions    {#sct_extension_types}
 
-IANA is asked to establish a registry of SCT extensions, named "Signed
-Certificate Timestamp Extension types" initially consisting of:
+IANA is asked to establish a registry of SCT extensions, named "CT Extension
+Types for SCT" initially consisting of:
```
Comment (robstradling) [inline draft-ietf-trans-rfc6962-bis.md]: There needs to be a comma before "initially".

I think "s/ initially consisting/, that initially consists/" would be better still.

File: draft-ietf-trans-rfc6962-bis.md [hunk 5]
```diff
@@ -1983,8 +1980,8 @@ All OIDs in the range from 1.3.101.8192 to 1.3.101.16383 have been reserved.
 This is a limited resource of 8,192 OIDs, each of which has an encoded length of
 4 octets.
 
-IANA is requested to establish a registry that will allocate Log IDs from this
-range.
+IANA is requested to establish a registry, named "CT Log ID Registry 1" that
+will allocate Log IDs from this range.
```
Comment (robstradling) [inline draft-ietf-trans-rfc6962-bis.md]: There needs to be a comma before "that".

File: draft-ietf-trans-rfc6962-bis.md [hunk 6]
```diff
@@ -1994,8 +1991,8 @@ The 1.3.101.80 arc has been delegated. This is an unlimited resource, but only
 the 128 OIDs from 1.3.101.80.0 to 1.3.101.80.127 have an encoded length of only
 4 octets.
 
-IANA is requested to establish a registry that will allocate Log IDs from this
-arc.
+IANA is requested to establish a registry, named "CT Log ID Registry 2" that
+will allocate Log IDs from this arc.
```
Comment (robstradling) [inline draft-ietf-trans-rfc6962-bis.md]: There needs to be a comma before "that".

File: draft-ietf-trans-rfc6962-bis.md (hunk 2 above)
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: Done.

File: draft-ietf-trans-rfc6962-bis.md (hunk 3 above)
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: Done.

File: draft-ietf-trans-rfc6962-bis.md (hunk 4 above)
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: Done.

File: draft-ietf-trans-rfc6962-bis.md [hunk 7]
```diff
@@ -1957,8 +1955,7 @@ TBD: policy for adding to the registry
 
 ## STH Extensions    {#sth_extension_types}
 
-IANA is asked to establish a registry of STH extensions, named "Signed Tree
-Head Extension types", initially consisting of:
+IANA is asked to establish a registry of STH extensions, named "CT Extension Types for STH", initially consisting of:
```
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: Done.

File: draft-ietf-trans-rfc6962-bis.md (hunk 5 above)
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: Done.

File: draft-ietf-trans-rfc6962-bis.md (hunk 6 above)
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: Done.

**Comment by eranmes** [comment]
@robstradling PTAL.

**Pull request merged**